test/node/conversation-labels.js

```diff
@@ -98,12 +103,29 @@ describe('Labels', () => {
         });
     });
 
+    it('should get conversations by the added label and check API returns the correct conversation', async () => {
+        sleep(3000);
+        const labelIds = Object.keys(addedLabelsHT);
+        const labelId = labelIds[0];
+        const res = await client.getConversationsByFilter({
+            filterConnector: {
+                conditions: [{
+                    filterTarget: Circuit.Constants.FilterTarget.LABEL_ID,
+                    expectedValue: [labelId]
+                }]
+            },
+            retrieveAction: Circuit.Enums.RetrieveAction.CONVERSATIONS
+        });
+        assert(res.find(conv => conv.convId === conversation.convId));
+
+    }).timeout(8000);
+
     it('should unassign the two labels from first conversation', async () => {
```

Comment:
'should unassign labels,

Let's keep the test description as simple and clear as possible